evento/src/components/TabContainer/index.js

```diff
@@ -1,23 +1,37 @@
 import React, { Component } from 'react';
-import { Link } from 'react-router-dom';
 import './TabContainer.css';
 
-const Tab = (props) => {
+const Tab = ({title, onClick, isSelected}) => {
+	const classname = 'Tab' + (isSelected ? ' selected' : '');
 	return (
-		<div className="Tab">
-			<h4>
-				<Link to={props.path}> {props.title} </Link>
-			</h4>
+		<div className={classname} onClick={() => onClick()}>
+			<h4> {title} </h4>
 		</div>
 	);
 };
 
 class TabContainer extends Component {
+	constructor(props) {
+		super(props);
+		const selected = this.props.tabs.filter((e) => e.path === props.path)[0];
+		this.state = { selected: selected };
+	}
+
+	onSelectedChange(tab) {
+		this.setState({ selected: tab });
+		this.props.onSelectedTabChange(tab.path);
```

Comment:
This should give tab as a parameter. TabContainer doesn't really even need to have any knowledge whether tab contains .path field or not, it's just additional data that may or may not be attached to the tab. Right now, this is not very reusable, since it assumes that all TabContainer always contains tabs which have .path and it doesn't give the tab object itself to the callback, just .path.